fearless/Common/EventCenter/EventVisitor.swift

```diff
@@ -14,6 +14,10 @@ protocol EventVisitorProtocol: AnyObject {
     func processChainSyncDidStart(event: ChainSyncDidStart)
     func processChainSyncDidComplete(event: ChainSyncDidComplete)
     func processChainSyncDidFail(event: ChainSyncDidFail)
+
+    func processRuntimeBaseTypesSyncCompleted(event: RuntimeBaseTypesSyncCompleted)
```

Comment:
I guess EventVisitor interface contains too many methods, can we loot to interface segregation?

Reply:
So the purpose of the EventCenter is to be the mediator for event delivery and visitor pattern helps to deal with many different types without switch...cases. But I will think how we could improve this if it grows abnormal.